### PR TITLE
T10 f22 67 implement comment service

### PR DESCRIPTION
**Pull request by ujjvalksharma**
implement comment service

### CODE REVIEW

**Review by hzhou74 — COMMENTED** [review]
ready to go

File: server.ts [hunk 1]
```diff
@@ -48,7 +48,7 @@ const likesController = LikeController.getInstance(app);
 const profileController = ProfileController.getInstance(app);
 const authorController = AuthenticationController(app);
 
-const PORT: any = process.env.PORT || 5000;
+const PORT: any = process.env.PORT || 5002;
```
Comment (Zevez7) [inline server.ts]: need to change this back to 5000

**Review by Zevez7 — APPROVED** [review]
This is an extra features that is not on the project plan.

File: server.ts (hunk 1 above)
Comment (ujjvalksharma) [reply server.ts]: Port changed back

**Pull request merged**